libs/designsystem/src/lib/components/item/item.component.scss

```diff
@@ -88,6 +88,14 @@
   }
 }
 
+.disclosure {
+  text-align: center;
+  height: 24px;
```

Comment:
Why hardcode a height here? I suspect this is not necessary.

Reply:
Removed